blog-content/monitoring-browser-applications-with-opentelemetry.md

```diff
@@ -37,15 +37,21 @@ These insights allow developers to identify bottlenecks, track user interactions
 
 Highlight's JavaScript SDK offers built-in support for collecting OpenTelemetry data from client-side applications. This integration allows you to seamlessly incorporate OpenTelemetry tracing into your web applications.
 
-Highlight automatically collects most of the OpenTelemetry data you'll need by leveraging the auto instrumentations and doing some additional processing to make the data more useful in Highlight. You can disable frontend traces by using the `disableOtelTracing` option as seen below
+Highlight automatically collects most of the OpenTelemetry data you'll need by leveraging the auto instrumentations and doing some additional processing to make the data more useful in Highlight. You can disable specific frontend traces by using the `otel.instrumentations` option as seen below:
 
 ```ts
 H.init({
   // ...
-  disableOtelTracing: true
+  otel: {
+   instrumentations: {
+      ['@opentelemetry/instrumentation-user-interaction']: false
+   }
+  }
 })
 ```
 
+This will disable any user interaction traces (e.g. mousemove) from being recorded.
+
```

Comment:
WDYT about adding a list + links of the instrumentations we enable by default?

Reply:
I can add those!